axum/src/extract/connect_info.rs

```diff
@@ -234,8 +219,91 @@ where
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::{routing::get, serve::IncomingStream, test_helpers::TestClient, Router};
-    use tokio::net::TcpListener;
+    use crate::{
+        extract::connect_info::Connected, routing::get, serve::IncomingStream, serve::Listener,
+        test_helpers::TestClient, Router,
+    };
+    use tokio::net::{TcpListener, TcpStream};
+
+    #[test]
+    fn into_make_service_traits() {
+        use crate::test_helpers::*;
+        assert_send::<IntoMakeServiceWithConnectInfo<(), NotSendSync>>();
+    }
+
+    #[allow(dead_code)]
+    #[test]
```

Comment:
```suggestion
    #[allow(clippy::todo)]
```

This doesn't need to be ran as a `#[test]`, we just want the compiler to compile it.